app/views/fear_quiz/new.html.erb

```diff
@@ -51,22 +51,22 @@
           <td>
             <%= check_box_tag 'box31', 'outdoors' %> outdoors <br>
             <%= check_box_tag 'box32', 'parents' %> parents <br>
-            <%= check_box_tag 'box33', 'rabbits' %> rabbits <br>
+            <%= check_box_tag 'error_404', '404' %> 404 errors <br>
```

Comment:
if a user checks this box and clicks enter it will redirect to a 404. stupid? yes.

Reply:
this is dumb. i love it.